.flaskenv

```diff
@@ -6,7 +6,7 @@ FLASK_APP=govuk-frontend-flask.py
 FLASK_DEBUG=True
 FLASK_RUN_PORT=5000
 REDIS_URL=memory://
-SECRET_KEY=
+SECRET_KEY= njndfnk342
```

Comment:
Could you remove this file, any env files should be kept for local development only. It might be good to add this to the gitignore.